src/annotator/test/notebook-test.js

```diff
@@ -1,6 +1,7 @@
 import Notebook from '../notebook';
 
 describe('Notebook', () => {
+  const sandbox = sinon.createSandbox();
```

Comment:
What is the need for the sandbox here? I assume it is because you're spying on methods on DOM objects? I don't think it is needed if you're spying on a method for a specific non-global DOM object. I would have assumed that the spy would be attached to the specific DOM object, which is created for a specific test, rather than by modifying the prototype.

Reply:
Ah, yes, I see; remedied.